tests/test_integrations_api.py

```diff
@@ -31,7 +31,7 @@ def test_send_msg_streaming(db_fixtures, force_authentication, mock_celery_tasks
     actual_events = []
     for event in r.text.split("\n\n"):
         assert not event.startswith("event: error"), event
-        data = event.split("data: ")[-1]
+        data = event.split("data: ", maxsplit=1)[-1]
```

Comment:
without this, another `data: ` somewhere in the event was being caught. we can also do `event.removeprefix("data: ")`